specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/examples/ClustersCreate.json

```diff
@@ -53,12 +41,7 @@
     "201": {
       "body": {
         "properties": {
-          "keyVaultProperties": {
-            "keyVaultUri": "https://aztest2170.vault.azure.net",
-            "keyName": "aztest2170cert",
-            "keyVersion": "654ft6c4e63845cbb50fd6fg51540429"
-          },
-          "provisioningState": "Succeeded",
+          "provisioningState": "Provisiong",
```

Comment:
I saw the "provisioningState" is "ProvisioningAccount" in response, will it be changed to "Provisioning"